File: app/src/main/java/com/rendersoncs/reportform/viewHolder/HeaderViewHolderExp.java
package com.rendersoncs.reportform.viewHolder;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import butterknife.BindView;
import butterknife.ButterKnife;
import com.rendersoncs.reportform.R;

class HeaderViewHolderExp extends RecyclerView.ViewHolder {
    @BindView(R.id.header_id)

    public TextView headerTitle;

    public HeaderViewHolderExp(@NonNull View itemView) {
        super(itemView);
        ButterKnife.bind(this, itemView);
    }
}

File: app/src/main/java/com/rendersoncs/reportform/view/adapter/viewHolder/HeaderViewHolderExp.kt
package com.rendersoncs.reportform.view.adapter.viewHolder

import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import butterknife.BindView
import butterknife.ButterKnife
import com.rendersoncs.reportform.R

internal class HeaderViewHolderExp(itemView: View) : RecyclerView.ViewHolder(itemView) {
    @JvmField
    @BindView(R.id.header_id)
    var headerTitle: TextView? = null

    init {
        ButterKnife.bind(this, itemView)
    }
}